sdk/servicebus/service-bus/src/core/linkEntity.ts

```diff
@@ -184,7 +184,7 @@ export class LinkEntity {
    * Ensures that the token is renewed within the predefined renewal margin.
    * @returns {void}
    */
-  protected async _ensureTokenRenewal(): Promise<void> {
+  protected _ensureTokenRenewal(): void {
```

Comment:
There are few places from where this is called. All those places need to be updated to remove the `await`